buda/entities/action_taken.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package buda;
+
+import "buda/entities/uuid.proto";
+import "google/protobuf/timestamp.proto";
+
+message ActionTaken {
```

Comment:
Do you think we could add some comments from your _Notes_ section code in this file? I can see them being helpful for future reference.

Reply:
Good idea @davidgasquez! Maybe going off Michael's idea in #19, we can use a semi-standard way to document our messages? https://github.com/pseudomuto/protoc-gen-doc#writing-documentation